### ISSUE TITLE
[quotes] False positive with backtick in require lines

### ISSUE THREAD

**Issue by regevbr** [typescript-eslint/typescript-eslint#1766]
**Repro**

```JSON
{
  "rules": {
        "@typescript-eslint/quotes": [
            "error",
            "backtick"
        ]
  }
}
```

```TS
import moment = require('moment');
```

**Expected Result**

Should not produce any error

**Actual Result**

produces an error, and fix will cause typescript compilation to fail as back ticks are not valid in a require statement.

**Additional info**
When you need to import from old modules that use the following syntax
```typescript
export = ...
```
You must use
```typescript
import moment = require('moment');
```
to import them, and the rule alerts on those.

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `^2.24.0` |
| `@typescript-eslint/parser`        | `^2.24.0` |
| `TypeScript`                       | `^3.8.3` |
| `ESLint`                           | `^6.8.0` |
| `node`                             | `v10.19.0` |
| `npm`                              | `6.13.4` |
| `moment`                              | `^2.24.0` |

**Related Issues**
#762
#757
#756

**Comment by bradzacher** [comment]
this is super easy to fix. Happy to accept a PR. It should only be a one line change in the rule, plus tests.

> You must use <import equals> to import them

Note that you don't actually _have_ to use that syntax, if you are using the `esModuleInterop` compiler option.

**Comment by regevbr** [comment]
Thanks I will try to give it a shot tomorrow
